File: backend/src/main/java/com/minetec/backend/dto/form/VehicleAttributeValueCreateUpdateForm.java
package com.minetec.backend.dto.form;

import lombok.Data;

import javax.validation.constraints.NotEmpty;
import javax.validation.constraints.NotNull;
import java.util.UUID;

@Data
public class VehicleAttributeValueCreateUpdateForm {

    @NotNull
    @NotEmpty
    private String desc;

    private UUID vehicleAttributeUuid;

    private UUID attributeValueUuid;


}

File: frontend/src/app/models/map/vehicle-attribute-value-create-update-form.ts

export class VehicleAttributeEntity {
   public uuid: string;
   public vehicleAttributeValue: string;
   public vehicleAttributeValueUuid: string;
}

export class VehicleAttributeValueCreateUpdateForm {

   public data: VehicleAttributeEntity;
   public desc: string;
   public vehicleAttributeUuid: string;
   public attributeValueUuid: string;
   public uuid: string;
   public vehicleAttributeValue: string;
}
